extensions/interactions/EndExploration/EndExploration.py

```diff
@@ -55,6 +55,10 @@ class EndExploration(base.BaseInteraction):
             'items': {
                 'type': 'unicode',
             },
+            'validators': [{
+                'id': 'has_length_at_most',
+                'max_value': 3,
+            }],
```

Comment:
Hmm, don't we need to do a validation of existing explorations on the server?

Reply:
Good q -- Here's how we're thinking about all of these frontend validation changes: we won't make any guarantees about the datastore, but are simply doing some initial frontend work before we do audit jobs and corresponding backend validation changes.

Reply:
Ok, is there a subsequent issue for tracking the required changes in the datastore and elsewhere?

Reply:
Yep, the issue is not posted yet, but will be in the future. For reference, see [this sheet](https://docs.google.com/spreadsheets/d/1kNLF-UkS_Qk0igNCGwLjJXM0ByHalqli41GPvWF43PE/edit?usp=sharing) for all validations (frontend + backend).

Reply:
Okay, looks good!